### PR TITLE
CL-1152 Add monitor carousel

### PR DESCRIPTION
**Pull request by ianpaschal**
It adds the thing.

Contributes to CL-1151

### CODE REVIEW

File: plugins/UM3NetworkPrinting/resources/qml/ExpandableCard.qml [hunk 1]
```diff
@@ -16,9 +16,9 @@ Item
 
     property bool expanded: false
     property var borderWidth: 1
-    property color borderColor: "#EAEAEC"
+    property color borderColor: "#CCCCCC"
```
[the remark this answers is not in the document]
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/ExpandableCard.qml]: Yes. Although I'm getting tired of writing `// TODO: Theme!`

Maybe I will just make a PR as soon as this one is merged theming everything so people stop asking me about it. 😛

File: plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        {
+            anchors.fill: parent
+            start: Qt.point(0, 0)
+            end: Qt.point(leftHint.width, 0)
+            gradient: Gradient
+            {
+                GradientStop
+                {
+                    position: 0.0
+                    color: "#fff6f6f6"
```
[the remark this answers is not in the document]
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml]: ^Ditto

File: plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+        {
+            anchors.fill: parent
+            start: Qt.point(0, 0)
+            end: Qt.point(rightHint.width, 0)
+            gradient: Gradient
+            {
+                GradientStop
+                {
+                    position: 0.0
+                    color: "#66f6f6f6"
```
[the remark this answers is not in the document]
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml]: ^Ditto

File: plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+                }
+                GradientStop
+                {
+                    position: 1.0
+                    color: "#fff6f6f6"
```
[the remark this answers is not in the document]
Comment (ianpaschal) [reply plugins/UM3NetworkPrinting/resources/qml/MonitorCarousel.qml]: Yeah. [You have QT to thank for that one.](http://doc.qt.io/qt-5/qml-color.html) 😵 🔫

**Comment by ChrisTerBeke** [comment]
Just tried it out, seems to work nicely. There's a weird grey dot now though if you have only one printer.

**Pull request merged**